api/src/main/java/com/capitalone/dashboard/rest/DeployController.java

```diff
@@ -50,4 +62,23 @@ public DeployController(DeployService deployService) {
                 .status(HttpStatus.CREATED)
                 .body(response);
     }
+    
+    @RequestMapping(value = "/deploy/rundeck", method = POST,
+            consumes = TEXT_XML_VALUE, produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> createRundeckBuild(HttpServletRequest request,
+            @RequestHeader("X-Rundeck-Notification-Execution-ID") String executionId, 
+            @RequestHeader("X-Rundeck-Notification-Trigger") String status) throws HygieiaException{
+        Document doc = null;
+        try {
+            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+            DocumentBuilder builder = factory.newDocumentBuilder();
+            doc = builder.parse(new InputSource(request.getInputStream()));
```

Comment:
@MarkRx I did experiment with creating a set of domain objects for the Rundeck POST and having Jackson unmarshall the XML to an object.  The domain options for XML unmarshalling through Jackson were quite complex and I ended up going with this implementation.

I can go back and write out domain objects again if you would like me to avoid doing the XML parsing explicitly.